src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -4,7 +4,8 @@
 
 public class ColorSupplier {
     public String getRandomColor() {
-        int index = new Random().nextInt(Colors.values().length);
-        return String.valueOf(Colors.values()[index]);
+        Colors[] colors = Colors.values();
+        int index = new Random().nextInt(colors.length);
+        return String.valueOf(colors[index]);
```

Comment:
or we can use `colors[index].name()` instead of `String.valueOf(colors[index])` - it's optional comment, you shouldn't change your code